src/openrct2/ride/RideConstruction.cpp

```diff
@@ -492,10 +492,10 @@ void ride_remove_provisional_track_piece()
     {
         int32_t flags = GAME_COMMAND_FLAG_APPLY | GAME_COMMAND_FLAG_ALLOW_DURING_PAUSED | GAME_COMMAND_FLAG_NO_SPEND
             | GAME_COMMAND_FLAG_GHOST;
-        maze_set_track(x, y, z, flags, false, 0, rideIndex, GC_SET_MAZE_TRACK_FILL);
-        maze_set_track(x, y + 16, z, flags, false, 1, rideIndex, GC_SET_MAZE_TRACK_FILL);
-        maze_set_track(x + 16, y + 16, z, flags, false, 2, rideIndex, GC_SET_MAZE_TRACK_FILL);
-        maze_set_track(x + 16, y, z, flags, false, 3, rideIndex, GC_SET_MAZE_TRACK_FILL);
+        maze_set_track(CoordsXYZD{ _unkF440C5, 0 }, flags, false, rideIndex, GC_SET_MAZE_TRACK_FILL);
```

Comment:
```suggestion
        maze_set_track(CoordsXYZD{ x, y, z, 0 }, flags, false, rideIndex, GC_SET_MAZE_TRACK_FILL);
```

I know why you did this, but this is more consistent.

Reply:
Yeah, I started with x, y, z and then realized I can use _unkF440C5, and then realized I can't really do that for the others, so I wasn't sure if I should keep it or not. I'll go back to x, y ,z. Thanks!